vip-jetpack/vip-jetpack.php

```diff
@@ -9,6 +9,34 @@
  * License: GPL2+
  */
 
+/** 
+ * Lowest incremental sync queue size allowed on VIP - JP default is 1000, but we're bumping to 10000 to give VIPs more
+ * headroom as they tend to publish more than average
+ */
+define( 'VIP_GO_JETPACK_SYNC_MAX_QUEUE_SIZE_LOWER_LIMIT', 10000 );
+
+/**
+ * The largest incremental sync queue size allowed - items will not get enqueued if there are already this many pending
+ * 
+ * The queue is stored in the option table, so if the queue gets _too_ large, site performance suffers
+ */
+define( 'VIP_GO_JETPACK_SYNC_MAX_QUEUE_SIZE_UPPER_LIMIT', 50000 );
```

Comment:
@lezama / @mdbitz what do you think about this upper limit? Should it be higher, perhaps 100k?